exo-clone/package.json

```diff
@@ -3,30 +3,29 @@
   "version": "0.0.6",
   "author": "Kevin Goslar",
   "dependencies": {
-    "async": "2.0.1",
+    "async": "^2.1.4",
```

Comment:
Remove the `^` everywhere here, and run `npm config set save-exact true` in the terminal to make NPM do the right thing here once and for all.